Gruntfile.js

```diff
@@ -138,78 +134,149 @@ module.exports = function(grunt) {
+                'Gruntfile.js', 
+                'ArticleTemplates/assets/js/{bootstraps, modules}/*.js'
+            ],
+            with_overrides: {
                 options: {
-                    run: false,
-                    log: true,
-                    urls: [ 'http://localhost:3000/root/test/unit/runner.html' ],
-                    page: {
-                        settings: {
-                            webSecurityEnabled: false,
-                        },
-                    },
+                    'bitwise': true,
+                    'browser': true,
+                    'camelcase': true,
+                    'curly': true,
+                    'eqeqeq': true,
+                    'expr': true,
+                    'forin': true,
+                    'immed': true,
+                    'indent': false,
+                    'latedef': false,
+                    'maxerr': 9999,
+                    'mocha': true,
+                    'newcap': true,
+                    'noarg': true,
+                    'noempty': true,
+                    'nonew': true,
+                    'quotmark': "single",
+                    'regexp': true,
+                    'strict': true,
+                    'trailing': true,
+                    'undef': true,
+                    'unused': true,
+                    'white': true,
+                    'predef': [ '-Promise' ],
+                    'globals': {
+                        'console': true,
+                        'GU': true,
+                        'require': true,
+                        'define': true,
+                        'sinon': true,
+                        'expect': true
+                    }
                 },
-            },
-            jenkins: {
+                files: {
+                    src: [
+                        'ArticleTemplates/assets/js/*.js',
+                        'ArticleTemplates/assets/js/bootstraps/article.js',
+                        'ArticleTemplates/assets/js/bootstraps/common.js',
+                        'test/spec/unit/**/*.js'
+                    ]
+                }
+            }
+        },
+        // unit tests
+        karma: {
+            unit: {
                 options: {
-                    run: false,
-                    log: true,
-                    reporter: 'XUnit',
-                    urls: [ 'http://localhost:3000/root/test/unit/runner.html' ],
-                    page: {
-                        settings: {
-                            webSecurityEnabled: false,
-                        },
+                    basePath: './',
+                    frameworks: ['mocha', 'requirejs', 'chai-sinon'],
+                    files: [
+                        {pattern: 'ArticleTemplates/assets/js/**/*.js' , included: false},
+                        {pattern: 'test/spec/unit/**/*.js', included: false},
+                        {pattern: 'node_modules/bonzo/bonzo.js', included: false},
```

Comment:
Can you use `node_modules/*` then explicitly include what you need (if anything)?